sql/sql_plugin.cc

```diff
@@ -1618,6 +1625,9 @@ bool plugin_register_builtin_and_init_core_se(int *argc, char **argv) {
   assert(global_system_variables.table_plugin);
   assert(global_system_variables.temp_table_plugin);
 
+  /* if ddse is rocksdb, rocksdb plugin should be loaded */
+  if (default_dd_storage_engine == DEFAULT_DD_ROCKSDB) assert(rocksdb_loaded);
```

Comment:
On release build the logic becomes unactionable, thus
```suggestion
assert(rocksdb_loaded || default_dd_storage_engine != DEFAULT_DD_ROCKSDB);
```